src/cbl-mariner/2.0/amd64/Dockerfile

```
@@ -0,0 +1,41 @@
+FROM mcr.microsoft.com/cbl-mariner/base/core:2.0
+RUN tdnf install -y \
+        ca-certificates \
+        autoconf \
+        automake \
+        build-essential \
+        clang \
+        cmake \
+        doxygen \
+        gcc \
+        gdb \
+        git \
+        krb5-devel \
+        libcurl-devel \
+        libedit-devel \
+        libicu-devel \
+        libmetalink-devel \
+        libnghttp2-devel \
+        libssh2-devel \
+        libtool \
+        libunwind-devel \
+        libuuid-devel \
+        libxml2-devel \
+        lttng-ust-devel \
+        lzma \
+        make \
+        ncurses-devel \
+        numactl-devel \
+        openssl-devel \
+	pkgconf \
+        readline-devel \
+        python3 \
+        python3-devel \
+	sed \
```

Making sure these are alphabetized.

```suggestion
	pkgconf \
        python3 \
        python3-devel \
        readline-devel \
	sed \
```